lib/experimental/Overlays/Tooltip/index.tsx

```diff
@@ -9,7 +9,7 @@ import {
 import React, { ComponentProps } from "react"
 
 interface TooltipProps {
-  label: string
+  label?: string
```

Comment:
something must be mandatory in a component, otherwise it gives too much flexibility to teams

What is the reason of removing this requirement?

Reply:
When we only want to specify a text with no bold

<img width="249" alt="Screenshot 2024-12-17 at 10 54 18" src="https://github.com/user-attachments/assets/c2152722-8059-496e-a35a-8158b6008000" />

Reply:
Sure!